Sources/protoc-gen-swiftgrpc/options.swift

```diff
@@ -131,6 +140,13 @@ final class GeneratorOptions {
           throw GenerationError.invalidParameterValue(name: pair.key, value: pair.value)
         }
 
+      case "ExtraModuleImports":
+        if !pair.value.isEmpty {
+          extraModuleImports.append(pair.value)
```

Comment:
Do we need to split the value here to handle multiple import statements? I.e., `pair.value.components(separatedBy: ",")`?

Reply:
No, per other comment multiple imports would be done by repeating the parameter.

`--swiftgrpc_opt=ExtraModuleImports=foo,ExtraModuleImports=qux --swiftgrpc_opt=ExtraModuleImports=baz`

The options are already comma separated in `parseParameter(string:)`, and changing the semantics there could break existing callers.